ehrapy/preprocessing/_normalization.py

```diff
@@ -8,9 +9,14 @@
 from ehrapy._compat import is_dask_array
 
 try:
+    import dask.array as da
     import dask_ml.preprocessing as daskml_pp
+
+    DASK_AVAILABLE = True
```

Comment:
What is your take on such a flag?
Could it be worth to introduce dask as a dependency now that we try to introduce it systematically?

Reply:
anndata[dask] I guess to be compatible but fine with me

Reply:
Yes, good point. If dask as dependency, then as `anndata[dask]`

Reply:
Depends on how slow dask is to import. This varies wildly depending on the project.

E.g. AnnData is a little slow because of pandas. In scanpy we lazy-import some sklearn subpackage because that would have a huge impact, and without it we’re only slower than anndata because of `sklearn.utils` and `numba`.